RESideMenu/RESideMenuController.m

```diff
@@ -651,32 +629,35 @@ - (void)hideMenuViewControllerAnimated:(BOOL)animated
         completionBlock();
     }
 
-    // Reset the perspective rotation and shadow animations
+    NSNumber *rotationFromValue = @(0.0);
+    if (isHidingRightMenu) {
+            rotationFromValue = @(-self.perspectiveRotationAmountRadians);
```

Comment:
Amazing how many of these kind of comments have just disappeared by using clang-format...